examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -12,11 +18,16 @@ import { renderAudience, renderDiceRoller } from "./view";
 // Define the server we will be using and initialize Fluid
 const useFrs = process.env.FLUID_CLIENT === "frs";
 
+const frsAzUser: FrsAzFuncUser = {
+    userId: "",
+    userName: "",
+};
+
 const user = generateUser();
```

Comment:
Could we have the frsAzUser object be filled out by what is returned by generateUser?